src/07 Day/categorize_numbers.py

```diff
@@ -0,0 +1,43 @@
+''' #7 day challenge:
+    Write a program to check if a number is positive, negative, or zero.
+
+
+    Output:
+    Categorize a list of numbers to positive, negative or zero:
+    [-7, -7, -6, -4, -2, -2, 0, 0, 0, 0, 1, 1, 5, 5, 7, 9, 10, 10, 10, 10]
+
+    Positive:
+    [1, 1, 5, 5, 7, 9, 10, 10, 10, 10]
+
+    Negative :
+    [-7, -7, -6, -4, -2, -2]
+
+    Zero:
+    [0, 0, 0, 0]
+'''
+import random
+
+def categorize_numbers(numbers):
+    ''' From the list create sublists with positive, negative and zero values '''
+    numbers.sort()
```

Comment:
There is no need to sort it

Reply:
Thank you for your suggestion. I will remove the sort.